src/components/ClaimMasterPanel.js

```diff
@@ -72,6 +72,11 @@ class ClaimMasterPanel extends FormPanel {
       "claimForm.claimTypeReferSymbol",
       'R',
     );
+    this.numberOfAdditionalDiagnosis = props.modulesManager.getConf(
+      "fe-claim",
+      "claimForm.numberOfAdditionalDiagnosis",
+      4,
```

Comment:
can you create const for this given value? It would be more readable for another developers to know exactly the meaning of value.